src/index.html

```diff
@@ -57,35 +57,13 @@ <h1 class="flex">What's Cookin, Good Lookin?</h1>
         </div>
       </section>
       <section class="flex row around main-display lrg-brdr-radius med-marg apricot shadow y-scroll" id="mainDisplay">
-        <article class="flex column sml-brdr-radius shadow">
```

Comment:
The satisfaction of deleting all the HTML after interpolating via a function = chefs kiss!